splash/examples/proxyEg.lua

```diff
@@ -0,0 +1,19 @@
+function main(splash)
+  splash:on_request(function(request)
+    request:set_proxy{
+        host = "10.3.100.207",
+        port = 8080,
+        username = splash.args.username,
+        password = splash.args.password,
+    }
+end)
+  local url = splash.args.url
+  assert(splash:go(url))
```

Comment:
It doesn't make sense to create a local variable and only use it once. Simply `assert(splash:go(splash.args.url))` would do.